conf/playReleases.json

```diff
@@ -1,11 +1,19 @@
 {
     "latest": {
-            "version": "2.8.6",
-            "date": "Dec 10 2020"
+            "version": "2.8.7",
+            "date": "Dec 14 2020"
     },
     "development": [
     ],
     "previous": [
+        {
+            "version": "2.7.9",
+            "date": "Dec 14 2020"
+        },
+        {
+            "version": "2.7.6",
```

Comment:
2.7.6? Sure? Was released in October and is further down in this list already.

Reply:
Thanks, fixed now.